src/components/ChannelPreview/ChannelPreviewMessage.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
Should reorder the imports and space out the component to match  the style of the others in the repo